File: solutions/0001. Two Sum/0001_1.cs
// Time complexity: O(n^2)
// Space complexity: O(1)
public class Solution {
    public int[] TwoSum(int[] nums, int target) {
        int[] V = { };
        for (int i = 0; i < nums.Length; i++) {
            for (int j = 0; j < nums.Length; j++) {
                if (i != j) {
                    if (nums[i] + nums[j] == target) {
                        return new int[] { i, j };
                    }
                }
            }
        }
        return V;
    }
}

File: solutions/0001. Two Sum/0001_1.py
# Time complexity: O(n^2)
# Space complexity: O(1)
class Solution:
    def twoSum(self, nums: list, target: int) -> list:
        for i in range(len(nums)):
            for j in range(i + 1, len(nums)):
                if nums[j] == target - nums[i]:
                    return [i, j] 